--- CMakeLists.txt	2025-04-30 13:57:07.843669876 +0530
+++ CMakeLists.txt.modulesaudio	2025-04-30 14:03:45.110385294 +0530
@@ -46,8 +46,10 @@
 set(SOLOUD_BACKEND_SDL2 OFF CACHE BOOL "")
 if (WIN32)
   set(SOLOUD_BACKEND_WINMM ON CACHE BOOL "")
-elseif (UNIX)
+elseif (UNIX AND NOT APPLE)
   set(SOLOUD_BACKEND_ALSA ON CACHE BOOL "")
+elseif (APPLE)
+  set(SOLOUD_BACKEND_COREAUDIO ON CACHE BOOL "")
 endif ()
 add_subdirectory(ext/soloud/contrib SYSTEM)
 
